nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/main/java/org/apache/nifi/processors/script/InvokeScriptedProcessor.java

```diff
@@ -92,11 +92,10 @@
                     logger.error(message, t);
                 }
             }
-        } else {
-            // Return defaults for now
-            relationships.add(REL_SUCCESS);
-            relationships.add(REL_FAILURE);
         }
+        // Add defaults
+        relationships.add(REL_SUCCESS);
+        relationships.add(REL_FAILURE);
```

Comment:
As per documentation of [ExecuteScript](https://github.com/apache/nifi/blob/1bd2cf0d09a7111bcecffd0f473aa71c25a69845/nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/main/java/org/apache/nifi/processors/script/ExecuteScript.java),  the script evaluation expects a FlowFile to be returned, in which case it will route the FlowFile to **success**. If a script error occurs, the original FlowFile will be routed to **failure**. 

So, to comply with that statement I think, it is required to add those defaults. 
Please correct me otherwise.